lib/components/Hidden/Hidden.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component, ReactNode, ReactType } from 'react';
+import Box from '../Box/Box';
+import styles from './Hidden.css.js';
+
+export interface HiddenProps {
+  children: ReactNode;
+  mobile?: boolean;
+  desktop?: boolean;
+  screen?: boolean;
+  print?: boolean;
+  inline?: boolean;
+  component?: ReactType;
```

Comment:
As discussed maybe remove the component option for now.

Reply:
Done 👍